README.md

```diff
@@ -191,13 +191,16 @@ Mathematical|trunc|![done][done]
 Mathematical|power, exp, nroot|![done][done]
 Mathematical|ln, log|![done][done]
 Mathematical|mod|![done][done]
+Mathematical|string_from_number|![done][done]
```

Comment:
Maybe we should put those in a separate table that is just for the methods that are out of the spec?